README.md

```diff
@@ -125,23 +128,65 @@ A single error is modeled to wrap the most common arguments of an error. Develop
 this model.
 
 ```kotlin
-class Error(
+@Serializable
+@SerialName("error")
+class DefaultError(
     val code: String,
     val title: String,
     val detail: String,
     val status: String
 )
 ```
 
-When using Retrofit, in the event of a network error a `HttpException` will be thrown. To extract the `Error` model from a response, you can
-use the `HttpException.asJsonXHttpException()` extension, which will return a `JsonXHttpException`, containing the original `response` as
-well as `errors` list.
+When using Retrofit, in the event of a network error a `HttpException` will be thrown. To extract the `DefaultError` model from a response, you can
+use the `HttpException.asJsonXHttpException<DefaultError>()` extension. The extension takes in a generic `DefaultError` which will return a `JsonXHttpException`, containing the original `response` as well as `errors` list.
```

Comment:
`asJsonXHttpException()` doesn't have a generic parameter to get the instance of `DefaultError`. Generic version of the extension is made separately and used only for custom errors

Reply:
Didn't believe you on this so had to check, will update. :)